app/assets/javascripts/discourse/components/global-notice.js.es6

```diff
@@ -17,6 +17,14 @@ export default Ember.Component.extend(StringBuffer, {
       notices.push([I18n.t("emails_are_disabled"), 'alert-emails-disabled']);
     }
 
+    if (Discourse.User.currentProp('staff') && this.siteSettings.bootstrap_mode_enabled) {
```

Comment:
We're still trying to deprecate constants in Javascript. You should use `this.currentUser && this.currentUser.get('staff')` instead.